src/m365/spo/commands/web/web-roleinheritance-break.ts

```diff
@@ -0,0 +1,69 @@
+import { Logger } from '../../../../cli';
+import GlobalOptions from '../../../../GlobalOptions';
+import SpoCommand from '../../../base/SpoCommand';
+import commands from '../../commands';
+import request from '../../../../request';
+import { validation } from '../../../../utils';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  webUrl: string;
+}
+
+class SpoWebRoleInheritanceBreakCommand extends SpoCommand {
+  public get name(): string {
+    return commands.WEB_ROLEINHERITANCE_BREAK;
+  }
+
+  public get description(): string {
+    return 'Break role inheritance of subsite';
+  }
+
+  constructor() {
+    super();
+
+    this.#initOptions();
+    this.#initValidators();
+  }
+
+  #initOptions(): void {
+    this.options.unshift(
+      {
+        option: '-u, --webUrl <webUrl>'
```

Comment:
If I check the specs, we are missing an option `-c --clearExistingPermissions`.

Reply:
We should also add a `--confirm` option to this command. This has been discussed later https://github.com/pnp/cli-microsoft365/issues/3593#issuecomment-1234828680.
Example of a `--confirm` flag can be found here: https://github.com/pnp/cli-microsoft365/blob/main/src/m365/planner/commands/bucket/bucket-remove.ts